### ISSUE TITLE
multi-gpu tutorial: batch size and optimization

### ISSUE THREAD

**Issue by aaronmarkham** [awslabs/keras-apache-mxnet#42]
When running the tutorial with the default settings it doesn't seem like the batch size is optimized for the # of gpus (and learning rate should be modified too), so maybe that should be discussed and set. When I tried to tinker with it, I wasn't able to really increase gpu utilization much.

```
ubuntu@ip-172-31-9-178:~$ watch -n0.1 nvidia-smi

Every 0.1s: nvidia-smi                                                                                       Fri Feb 16 23:00:41 2018

Fri Feb 16 23:00:41 2018
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 384.111                Driver Version: 384.111                   |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Tesla V100-SXM2...  On   | 00000000:00:1B.0 Off |                    0 |
| N/A   44C    P0    53W / 300W |    786MiB / 16152MiB |     15%      Default |
+-------------------------------+----------------------+----------------------+
|   1  Tesla V100-SXM2...  On   | 00000000:00:1C.0 Off |                    0 |
| N/A   41C    P0    51W / 300W |    786MiB / 16152MiB |     17%      Default |
+-------------------------------+----------------------+----------------------+
|   2  Tesla V100-SXM2...  On   | 00000000:00:1D.0 Off |                    0 |
| N/A   41C    P0    53W / 300W |    786MiB / 16152MiB |     18%      Default |
+-------------------------------+----------------------+----------------------+
|   3  Tesla V100-SXM2...  On   | 00000000:00:1E.0 Off |                    0 |
| N/A   43C    P0    55W / 300W |    786MiB / 16152MiB |     19%      Default |
+-------------------------------+----------------------+----------------------+

+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      2774      C   python                                       776MiB |
|    1      2774      C   python                                       776MiB |
|    2      2774      C   python                                       776MiB |
|    3      2774      C   python                                       776MiB |
+-----------------------------------------------------------------------------+
```

**Comment by sandeep-krishnamurthy** [comment]
Since the input was CIFAR10 (32*32) images, which is very tiny, GPU utilization is very low. We tried with ImageNet dataset and saw an effect of increasing GPU utilization and batch size. 
After this PR is merged, you can try with ImageNet dataset - https://github.com/awslabs/keras-apache-mxnet/pull/55

Please reopen the issue, if you find similar behavior with larger dataset.
